posts/inside-rust/2020-10-08-regression-labels.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Marking issues as regressions"
+description: "Now anyone can mark issues as regressions!"
+author: Camelid
+team: the release team <https://www.rust-lang.org/governance/teams/operations#release>
+---
+
+If you've contributed to Rust before, you have likely used a command like this
+to [set labels][rustbot-labeling-docs]:
+
+> @**rustbot** modify labels: A-typesystem C-enhancement
```

Comment:
This is the pickiest nitpick possible: I think on GitHub the `@` sign before a mention is bold too
```suggestion
> **@rustbot** modify labels: A-typesystem C-enhancement
```

Reply:
why don't we just do `@rustbot modify ...` which is what the user writes?

Reply:
This is markdown, which will be rendered as `@rustbot modify ...` anyway, so I don't think it matters much.
I personally prefer it with the bold text, so it makes clear to the reader who's used to GitHub that is is a mention, but, 🤷

Reply:
You’re right that that’s how GitHub renders it. I was kind of going off how you type a mention in Zulip.